readme.md

```diff
@@ -0,0 +1,88 @@
+# Ukrainianconflict-live
+
+This project aims to provide a live feed about Ukrainian invasion.
+
+## Technical choices
+
+For the sake of security and efficacity, time is a challenge here, and this website 
+has to be bulletproof as much as possible.
+
+We made the following choices so far:
+
+#### No JS  
+
+Website shall be accessible via TOR or by users who disabled JS on their browser
+
+#### No framework
+
+Time is a challenge here, no time to learn somtehing common accross all of the volunteers.
+
+#### Mobile first
+
+Website has to be designed mobile first, since it is the most used terminal used by the one who 
+need it.
+
+## Setup the project
+
+First, install python 3.8+: <https://www.python.org/downloads/release/python-3910/>
+
+Then run in the project directory:
+
+    pip install -r requirement.txt
```

Comment:
```suggestion
    pip install -r requirements.txt
```

Reply:
thanks, god I was really tired yesterday night to make such mistakes. I'll fix it soon